docs/Configuration.md

```diff
@@ -50,8 +53,18 @@
 - **totalPriceStatus** - The status of the total price used. Defaults to `"FINAL"`.
 - **allowPrepaidCards** - Set to `false` if you don't support prepaid cards. Default: The prepaid card class is supported for the card networks specified.
 - **billingAddressRequired** - Set to `true` if you require a billing address. A billing address should only be requested if it's required to process the transaction.
+- **billingAddressParameters** - Set billing address parameters:
+
+  - **format** - Billing address format required to complete the transaction. _MIN_: Name, country code, and postal code (default)._FULL_: Name, street address, locality, region, country code, and postal code.
```

Comment:
The name of this name, **format**, seems a like the wrong word for it. It doesn't necessarily imply the fields that are included in the address. Maybe another name and description would be more clear?

Name suggestion: **requiredFields**.  Description suggestion: The address fields required to complete the transaction.

Are _MIN_ and _FULL_ the possible values? If so, it would be more clear to say it like this:

Possible values:
- **MIN** (default):  Name, country code, and postal code.
- **FULL**: Name, street address, locality, region, country code, and postal code.

Reply:
"format" is [Google's](https://developers.google.com/pay/api/android/reference/request-objects#BillingAddressParameters) naming 🤷‍♂️

Rest, fixed in [9532e80](https://github.com/Adyen/adyen-react-native/pull/297/commits/9532e80dd10c9f43f8b43db67dbe84b52fc2a5d8)